src/pages/SettingsPage.js

```diff
@@ -94,6 +120,29 @@ const SettingsPage = ({
                             Sign Out
                         </Text>
                     </TouchableOpacity>
+                    <View style={[styles.settingsPageBody, styles.mt6]}>
+                        <Text style={[styles.settingsPageHeading, styles.mt2, styles.mb4]} numberOfLines={1}>
+                            Chats
+                        </Text>
+                        <Text style={[styles.textP]}>
+                            How should we display chats on your home screen?
+                        </Text>
+                        <View style={[styles.mt2, styles.mb2]}>
+                            <RNPickerSelect
+                                onValueChange={updatePriorityMode}
+                                items={Object.values(priorityModes)}
+                                style={{
+                                    ...styles.picker,
+                                }}
+                                useNativeAndroidPickerStyle={false}
+                                placeholder={{}}
+                                Icon={() => <Icon src={DownArrow} />}
+                            />
+                        </View>
+                        <Text style={[styles.textP, styles.colorMuted]}>
+                            {priorityModes[priorityMode]?.description }
```

Comment:
Sorry, we don't really use optional chaining so this needs to be done an alternate way. Separately, not sure I see the case where this wouldn't have a value or what we're trying to avoid. If there was such a case we'd then still be returning a `Text` component with no contents?

Reply:
Sorry I added this when testing as the app wouldn't recover from a crash when the cause was persisted by ONXY giving an invalid priority mode in props and therefore making an invalid reference here. It shouldn't be needed in prod code.
good catch thank you!